doc/neps/nep-0029-deprecation_policy.rst

```diff
@@ -150,7 +153,10 @@ Drop Schedule
   On Jun 23, 2023 drop support for NumPy 1.21 (initially released on Jun 22, 2021)
   On Jan 01, 2024 drop support for NumPy 1.22 (initially released on Dec 31, 2021)
   On Apr 05, 2024 drop support for Python 3.9 (initially released on Oct 05, 2020)
+  On Jun 22, 2024 drop support for NumPy 1.23 (initially released on Jun 22, 2022)
```

Comment:
I note that previous entries dropped numpy support a day later than the month/day of release. I don't think it matters, just noting.